### DISCUSSION TITLE
Could not resolve schema.prisma in Deploying with AWS SAM

### DISCUSSION THREAD

**Discussion by koSakano** [prisma/prisma#21468] [Q&A]
### Question

When I prepared lambda by AWS SAM according to [the Documentation](https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-aws-lambda?query=libquery_engine-rhel-openssl-1.0.x.so.node&page=1#deploying-with-aws-sam), the following error occurred.
```
// Could not resolve "./node_modules/.prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node"
import x from './node_modules/.prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node';
```

What steps can be taken to resolve this?

### How to reproduce (optional)

1 Set up AWS SAM environment by [this repository](https://github.com/aws/aws-sam-cli-app-templates/tree/master/nodejs18.x/hello-ts/%7B%7Bcookiecutter.project_name%7D%7D)
2 Add prisma by [this](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases-typescript-postgresql)
3 Change settings by [this](https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-aws-lambda?query=libquery_engine-rhel-openssl-1.0.x.so.node&page=1#deploying-with-aws-sam)

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @koSakano 👋 

Thank you for raising this question.

Did you define the `binaryTargets` in the generator block like [this](https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-aws-lambda#define-binary-targets-in-prisma-schema)?
```
binaryTargets = ["native", "rhel-openssl-1.0.x"]
```

**Comment by koSakano** [comment]
Hi @ludralph 
Yes.My `schema.prisma` file generate `client` as follows.
```prisma:schema.prisma
generator client {
  provider = "prisma-client-js"
  binaryTargets = ["native", "rhel-openssl-1.0.x"]
}
```

**Comment by koSakano** [comment]
@ludralph 
Sorry.The cause was not known, but it was fixed after re-setting up the system.
